playbooks/sudocmdgroup/ensure-sudocmd-are-absent-in-sudocmdgroup.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Please rename the file to `ensure-sudocmdgroups-are-absent-in-sudocmdgroup.yml `